### ISSUE TITLE
produce a start to finish example including model and obs data, parameter file and creation of portrait plot

### ISSUE THREAD

**Issue by gleckler1** [PCMDI/pcmdi_metrics#343]
@doutriaux1 lets team up onthis today, get it in there, and then tag v1.1

**Comment by durack1** [comment]
@gleckler1 I suggest moving this to `1.2`

**Comment by durack1** [comment]
@gleckler1 this needs to be merged before #374 can be finalised - as you're generating files and directory structures that `setup.py` needs to know about.. So unless you can specify what stuff needs to be updated in `setup.py` and the `conda install` path this will need to be finalised before #374 and then #375 tagging and release step..

**Comment by gleckler1** [comment]
@durack1 changes to setup.py have been proposed in pr https://github.com/PCMDI/pcmdi_metrics/pull/376

**Comment by gleckler1** [comment]
@doutriaux1  It would be nice if we could point to a complete list of "official keys" available for the PMP.   I think that is documented (sort of) in input_parameter_file_all_options.py which is in the /doc directory of the repo.   I guess We should point to this file in the repo rather than its buried location in the build?

**Comment by durack1** [comment]
@gleckler1 @doutriaux1 this is still open, AFTER tagging `1.1.1` is it to be moved to another milestone or is it already done?

**Comment by durack1** [comment]
@gleckler1 it's also `HIPRIORITY` which means it could be more or less important than `CRITICAL` or `TOPPRIORITY`, yet may be less important (or is it more important) than `DECISIONNEEDEDNOW`, `Lingering but still important` or `SolutionUnderTesting`

**Comment by doutriaux1** [comment]
@durack1 enjoy your vacation. `pmp_demo_1.py` does this up to the portrait creation. The branch `jason_class` is a HUGE improvement that will make the last part (portraits creation) much easier for the end user.

**Comment by durack1** [comment]
@doutriaux1 then do you want to close this, and we can tweak under a different issue/milestone?

**Comment by doutriaux1** [comment]
@durack1 opened #390 to finish this
